Tests/Common/Util/ExtensionsTests.cs

```diff
@@ -234,8 +237,8 @@ public void ConvertsDictionaryFromString()
         public void DictionaryAddsItemToExistsList()
         {
             const int key = 0;
-            var list = new List<int> {1, 2};
-            var dictionary = new Dictionary<int, List<int>> {{key, list}};
+            var list = new List<int> { 1, 2 };
+            var dictionary = new Dictionary<int, List<int>> { { key, list } };
```

Comment:
#### No Action Required.
When possible, try to avoid these types of white space changes. I know my settings are configured to format like the left w/out spaces following `{` -- so changes like these often just cause code churn and diffs noisy. I guess my point is, since we don't have a full-featured style guide, we shouldn't be updating existing code styles that are in line with the existing conventions.

I have a lower priority to-do items on my list for improving the R# `dotSettings` files w.r.t code style, so hopefully in the future it's as easy as telling R# to format it according to the `dotSettings` file and then we're all consistent with each other. easy peasy.